src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -285,37 +287,45 @@ public void parse_optionalFieldMissing_success() {
 
         assertParseSuccess(parser,
             NAME_DESC_BOB + EMAIL_DESC_BOB + ADDRESS_DESC_BOB + GENDER_DESC_BOB + IC_NUMBER_DESC_BOB + BIRTHDAY_DESC_BOB
-                + TAG_DESC_HUSBAND + TAG_DESC_FRIEND, new AddCommand(expectedPatient));
+                + PRIORITY_DESC_BOB + TAG_DESC_HUSBAND + TAG_DESC_FRIEND, new AddCommand(expectedPatient));
 
         // optional field Email missing (ie default email used)
         expectedPatient = new PatientBuilder(BOB).withEmail(Email.getDefaultEmail()).build();
 
         assertParseSuccess(parser,
             NAME_DESC_BOB + PHONE_DESC_BOB + ADDRESS_DESC_BOB + GENDER_DESC_BOB + IC_NUMBER_DESC_BOB + BIRTHDAY_DESC_BOB
-                + TAG_DESC_HUSBAND + TAG_DESC_FRIEND, new AddCommand(expectedPatient));
+                + PRIORITY_DESC_BOB + TAG_DESC_HUSBAND + TAG_DESC_FRIEND, new AddCommand(expectedPatient));
 
 
         // optional field address missing (ie default gender used)
         expectedPatient = new PatientBuilder(BOB).withGender(Gender.getDefaultGender()).build();
 
         assertParseSuccess(parser,
             NAME_DESC_BOB + EMAIL_DESC_BOB + PHONE_DESC_BOB + ADDRESS_DESC_BOB + IC_NUMBER_DESC_BOB + BIRTHDAY_DESC_BOB
-                + TAG_DESC_HUSBAND + TAG_DESC_FRIEND, new AddCommand(expectedPatient));
+                + PRIORITY_DESC_BOB + TAG_DESC_HUSBAND + TAG_DESC_FRIEND, new AddCommand(expectedPatient));
 
 
         // optional field address missing (ie default birthday used)
         expectedPatient = new PatientBuilder(BOB).withBirthday(Birthday.getDefaultBirthday()).build();
 
         assertParseSuccess(parser,
             NAME_DESC_BOB + EMAIL_DESC_BOB + PHONE_DESC_BOB + ADDRESS_DESC_BOB + GENDER_DESC_BOB + IC_NUMBER_DESC_BOB
-                + TAG_DESC_HUSBAND + TAG_DESC_FRIEND, new AddCommand(expectedPatient));
+                + PRIORITY_DESC_BOB + TAG_DESC_HUSBAND + TAG_DESC_FRIEND, new AddCommand(expectedPatient));
 
         // optional field address missing (ie default address used)
         expectedPatient = new PatientBuilder(BOB).withAddress(Address.getDefaultAddress()).build();
 
         assertParseSuccess(parser,
             NAME_DESC_BOB + EMAIL_DESC_BOB + PHONE_DESC_BOB + GENDER_DESC_BOB + IC_NUMBER_DESC_BOB + BIRTHDAY_DESC_BOB
-                + TAG_DESC_HUSBAND + TAG_DESC_FRIEND, new AddCommand(expectedPatient));
+                + PRIORITY_DESC_BOB + TAG_DESC_HUSBAND + TAG_DESC_FRIEND, new AddCommand(expectedPatient));
+
+        // optional field priority missing (ie default priority used)
+        expectedPatient = new PatientBuilder(BOB).withPriority(Priority.getDefaultPriority()).build();
+
+        assertParseSuccess(parser,
+                NAME_DESC_BOB + EMAIL_DESC_BOB + PHONE_DESC_BOB + GENDER_DESC_BOB + IC_NUMBER_DESC_BOB
+                        + BIRTHDAY_DESC_BOB + ADDRESS_DESC_BOB + TAG_DESC_HUSBAND + TAG_DESC_FRIEND,
+                new AddCommand(expectedPatient));
     }
```

Comment:
Nice job following the agenda of that test case with the new priority attribute!